theme/README.md

```diff
@@ -1,51 +1,139 @@
-# Gatsby Theme Jam Example Submission
+<p align="center">
+  <a href="https://www.gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  @olavea/gatsby-theme-book
+</h1>
 
-This is a bare-bones Gatsby theme to showcase how a [Theme Jam](https://themejam.gatsbyjs.org) submission should look.
+## A Gatsby theme for building a book.
 
-See the [live demo](https://gatsby-theme-jam-example.netlify.com)
+I love reading books. So the day it dawned on me that I could build a book with Gatsby was a GOOD day. I will try to pass some of the goodness of that day to you with this Gatsby Book Theme.
+
+If you want to read how my daughter Lillian (4) gave me this book building idea see below.
+
+[**Demo Website**](https://petra-kanini.netlify.com)
+
+## Features
+-
+-
+-
 
 ## Installation
 
-To use this theme in your Gatsby sites, follow these instructions:
+```sh
+npm install @olavea/gatsby-theme-book
+```
+
+### Install as a starter
+
+This will generate a new book (with the folder name "book") that pre-configures use of the theme including example content.
+
+```sh
+gatsby new book LekoArts/gatsby-starter-book
+```
+
+## Usage
 
-1.  Install the theme
-    ```sh
-    npm install --save gatsby-theme-jam-example
-    ```
+### Theme options
 
-2.  Add the theme to your `gatsby-config.js`:
-    ```js
-    module.exports = {
-      plugins: [
-        'gatsby-theme-jam-example'
-      ]
+| Key            | Default Value      | Description                                                                                               |
+| -------------- | ------------------ | --------------------------------------------------------------------------------------------------------- |
+| `basePath`     | `/`                | Root url for the theme                                                                                    |
+| `imagePath` | `book/pics` | Location of picturess                                                                                      |
```

Comment:
Either change the default in `gatsby-config.js` or change it here to `book/images`.